multinet/__init__.py

```diff
@@ -21,13 +21,48 @@
 sentry_sdk.init(dsn=sentry_dsn, integrations=[FlaskIntegration()])
 
 
-def get_allowed_origins() -> List[str]:
+def get_allowed_origins() -> Set[str]:
     """Read in comma-separated list of allowed origins from environment."""
     allowed_origins = os.getenv("ALLOWED_ORIGINS", default=None)
     if allowed_origins is None:
-        return []
+        return set()
```

Comment:
I think flake8 may complain about this line.

```suggestion
        return {}
```

Reply:
You actually need to use `set()` to initialize an empty set, as `{}` represents an empty dictionary.

Reply:
Good point.